### PR TITLE
[SDK] Enhance Operator Data

### LINKED ISSUE #1835: SDK - Add properties to response of getReputationNetworkOperators
**Description**
Add the value of the following keys from kvstore to object returned in getReputationNetworkOperators:
- "url"
- "job_types"

Implement it on both TS and Python

**Motivation**
Make possibel the oracles discovery in one request

### PR DESCRIPTION
**Pull request by mrhouzlane**
## Description

Enhance the response of `getReputationNetworkOperators` by including additional properties from the KV store: "url" and "job_types". 
- This enhancement will facilitate more comprehensive oracle discovery with a single request

#### TypeScript  : 

- Modify `getReputationNetworkOperators` to fetch and append `url` and `job_types` properties for each operator from the KV store. 
- url is optional and job_types non-optional. 

#### Python : 

Closes #1835

### CODE REVIEW

File: packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts [hunk 1]
```diff
@@ -48,6 +48,8 @@ export interface IReputationNetwork {
 export interface IOperator {
   address: string;
   role?: string;
+  url: string;
```
Comment (portuu3) [inline packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts:51]: this should be nullable

File: packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts [hunk 2]
```diff
@@ -48,6 +48,8 @@ export interface IReputationNetwork {
...
+  jobTypes: string;
```
Comment (portuu3) [inline packages/sdk/typescript/human-protocol-sdk/src/interfaces.ts:52]: This should be nullable.
Lets make jobTypes a string[], use split function

**Review by portuu3 — CHANGES_REQUESTED** [review]
Apply the same comments to python.
You should also regenerate the python docs: `make generate-doc-md` in python sdk folder

File: packages/apps/job-launcher/server/test/setup.ts [hunk 3]
```diff
@@ -31,9 +31,28 @@ export async function setup(): Promise<void> {
...
+      KVStoreKeys.url,
+      KVStoreKeys.jobTypes,
+    ],
+    [
+      Role.JobLauncher,
+      '1',
+      'http://localhost:5000/webhook',
+      'http://localhost:5000',
+      jobTypesString,
+    ],
```
Comment (portuu3) [inline packages/apps/job-launcher/server/test/setup.ts:55]: Keep it simple
```suggestion
  await kvStoreClient.setBulk(
    [
      KVStoreKeys.role,
      KVStoreKeys.fee,
      KVStoreKeys.webhookUrl,
      KVStoreKeys.url,
      KVStoreKeys.jobTypes,
    ],
    [
      Role.JobLauncher,
      '1',
      'http://localhost:5000/webhook',
      'http://localhost:5000',
      'type1,type2',
    ],
```

File: packages/sdk/python/human-protocol-sdk/test/human_protocol_sdk/operator/test_operator_utils.py [hunk 4]
```diff
@@ -181,6 +181,8 @@ def test_get_reputation_network_operators(self):
         reputation_address = "0x1234567890123456789012345678901234567891"
         operator_address = "0x1234567890123456789012345678901234567891"
         role = "Job Launcher"
+        url = "https://example.com"
+        job_types = ["type1, type2"]
```
Comment (portuu3) [inline packages/sdk/python/human-protocol-sdk/test/human_protocol_sdk/operator/test_operator_utils.py:185]:
```suggestion
        job_types = ["type1", "type2"]
```

**Review by portuu3 — CHANGES_REQUESTED** [review]

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py [hunk 5]
```diff
@@ -216,7 +216,7 @@ class ChainId(Enum):
     ChainId.LOCALHOST: {
         "title": "Localhost",
         "scan_url": "",
-        "subgraph_url": "subgraph_url",
```
Comment (mrhouzlane) [inline packages/sdk/python/human-protocol-sdk/human_protocol_sdk/constants.py:219]: do we have to make this change ?

**Review by portuu3 — APPROVED** [review]

**Pull request merged**